tests/test_jinja_macros.py

```diff
@@ -41,7 +41,7 @@ def default_flask_g():
         full_name="Test Lead Assessor",
         email="test@example.com",
         roles=["COF_LEAD_ASSESSOR", "COF_ASSESSOR", "COF_COMMENTER"],
-        highest_role="COF_LEAD_ASSESSOR",
+        highest_role_map="COF_LEAD_ASSESSOR",
```

Comment:
This should maybe be a map, but it's not used anymore so might be able to remove the g.user from this test all together?

Reply:
User class has an attribute "highest_role_map" in utils?

Reply:
I would rather have a conversation with the person who has been working on this functionality but my focus is more on the functionality I have been working on related to mapping and formatting.